package.json

```diff
@@ -5,20 +5,24 @@
   "main": "index.js",
   "scripts": {
     "start": "parallelshell \"node web/index.js\" \"npm run watch:css\" \"npm run watch:js\"",
-    "test": "npm run test:server && npm run test:browser && npm run lint",
+    "start:production": "parallelshell \"node web/index.js\" \"npm run watch:css\" \"npm run watch:js:production\"",
     "build": "npm run build:css && npm run build:js",
+    "build:production": "npm run build:css && npm run build:js:production",
     "build:css": "npm run watch:css -- --no-watch",
     "build:js": "webpack --config webpack.config.js --progress --profile --colors",
+    "build:js:production": "webpack --config webpack.config.js --progress --profile --colors  --optimize-minimize --optimize-dedupe",
```

Comment:
You could change this to `npm run build:js -- --optimize-minimize --optimize-dedupe`